lms/djangoapps/mobile_api/social_facebook/courses/tests.py

```diff
@@ -11,6 +14,7 @@
 from ..test_utils import SocialFacebookTestCase
 
 
+@patch.dict(settings.FEATURES, {"CUSTOM_COURSES_EDX": False})
```

Comment:
It is enabled by default and is_enabled tries to get custom course because of `if settings.FEATURES.get('CUSTOM_COURSES_EDX', False)` check present in DiscussionTab `is-enabled` function.